[BUG] Panic in `terrad export`

**Describe the bug**
`terrad export --height XXX` panics.

**To Reproduce**
`terra terrad export --height=5257270 > terra_snapshot.json`.

Results in the following output:
```
panic: Null/Nil argument: checksum

goroutine 1 [running]:
github.com/terra-money/core/x/wasm.ExportGenesis({{0x23641e8, 0xc000052048}, {0x23914e0, 0xc01658a200}, {{0x0, 0x0}, {0x0, 0x0}, 0x503836, {0x0, ...}, ...}, ...}, ...)
	github.com/terra-money/core/x/wasm/genesis.go:67 +0x799
github.com/terra-money/core/x/wasm.AppModule.ExportGenesis({{{0x2397bf8, 0xc0012c4e30}}, {{0x232f070, 0xc001e4dd20}, {0x238c518, 0xc0012c4e30}, {{0x238c518, 0xc0012c4e30}, 0xc000011d28, {0x232f070, ...}, ...}, ...}, ...}, ...)
	github.com/terra-money/core/x/wasm/module.go:158 +0x9b
github.com/cosmos/cosmos-sdk/types/module.(*Manager).ExportGenesis(_, {{0x23641e8, 0xc000052048}, {0x23914e0, 0xc01658a200}, {{0x0, 0x0}, {0x0, 0x0}, 0x503836, ...}, ...}, ...)
	github.com/cosmos/cosmos-sdk@v0.44.2/types/module/module.go:328 +0x125
github.com/terra-money/core/app.(*TerraApp).ExportAppStateAndValidators(0xc001227b00, 0x0, {0x3302f78, 0x0, 0x0})
	github.com/terra-money/core/app/export.go:34 +0x20f
main.appCreator.appExport({{{0x236d6f8, 0xc0012ca390}, {0x2397bf8, 0xc0012c4e30}, {0x237ffb8, 0xc00122d100}, 0xc000011d28}}, {0x2364e98, 0xc001435aa0}, {0x238d218, ...}, ...)
	github.com/terra-money/core/cmd/terrad/root.go:255 +0x44a
github.com/cosmos/cosmos-sdk/server.ExportCmd.func1(0xc00131d680, {0xc0013b52f0, 0x0, 0x1})
	github.com/cosmos/cosmos-sdk@v0.44.2/server/export.go:71 +0x31c
github.com/spf13/cobra.(*Command).execute(0xc00131d680, {0xc0013b52e0, 0x1, 0x1})
	github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc001234280)
	github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.2.1/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v1.2.1/command.go:895
github.com/cosmos/cosmos-sdk/server/cmd.Execute(0x0, {0xc001231d70, 0xd})
	github.com/cosmos/cosmos-sdk@v0.44.2/server/cmd/execute.go:36 +0x1eb
main.main()
	github.com/terra-money/core/cmd/terrad/main.go:15 +0x2c
```

**Context & versions**
terrad 0.5.10 and 0.5.11

--- a/x/wasm/genesis.go
+++ b/x/wasm/genesis.go
@@ -62,14 +62,17 @@ func ExportGenesis(ctx sdk.Context, keeper keeper.Keeper) *types.GenesisState {
 	}
 
 	for i := uint64(1); i <= lastCodeID; i++ {
-		bytecode, err := keeper.GetByteCode(ctx, i)
+		codeInfo, err := keeper.GetCodeInfo(ctx, i)
 		if err != nil {
 			panic(err)
 		}
 
-		codeInfo, err := keeper.GetCodeInfo(ctx, i)
-		if err != nil {
-			panic(err)
+		var bytecode []byte
+		if len(codeInfo.CodeHash) != 0 {
+			bytecode, err = keeper.GetByteCode(ctx, i)
+			if err != nil {
+				panic(err)
+			}
 		}
 
 		codes = append(codes, types.Code{

